recipes/gitlab/webview.js

```diff
@@ -0,0 +1,35 @@
+module.exports = Ferdium => {
+    const getMessages = () => {
+      let direct = 0;
+      let indirect = 0;
+      const elements = document.querySelectorAll('.rp');
```

Comment:
Are these badge counters valid? Have they been verified?

Reply:
I haven't verified the badge counters given that I don't use this recipes. I just followed the sample_recipe and added those. I'll remove all so we can merge this PR and then, if needed, add it.